### ISSUE TITLE
feat: Voice Recognition stops listening after 1 silence second, can i increase it ?

### ISSUE THREAD

**Issue by Labaffe** [capacitor-community/speech-recognition#102]
all is title

**Comment by tobika** [comment]
Hello,
I have the same problem. The google prompt is closing just after 1 sec of silenec. Adding this parameters doesn't seem to have an effect:
```
intent.putExtra(RecognizerIntent.EXTRA_SPEECH_INPUT_MINIMUM_LENGTH_MILLIS, Long.valueOf(5000));
intent.putExtra(RecognizerIntent.EXTRA_SPEECH_INPUT_POSSIBLY_COMPLETE_SILENCE_LENGTH_MILLIS, Long.valueOf(5000));
```

For info I am using the google prompt, not the web implementation.

**Comment by tobika** [comment]
I found a solution that works without the native google prompt. Will propose a PR in the coming days :)

**Comment by wgehner** [comment]
Yes, the listeningState event is firing the the stopped event way too quickly. I need to extend it to know when to submit the data to ai. Not using prompt.

**Comment by 222dipa** [comment]
Hi @tobika, I downloaded your fork project, compiled and installed it in my Ionic project. 

How can I use the allowForSilence property?

I'm trying to set 60000ms, but after about 20 seconds of silence the microphone automatically closes. 

Also, could you tell me how to set the partialResults property? Should it be true or false? Where is it best to collect the results? Adding a listener for partialResults or segmentResults?

Thanks so much!!!!

**Comment by msty** [comment]
Have the same problem on android. on iOS the issue doesn't seem to be present

**Comment by MalikHamza007** [comment]
> Can confirm I observe the same issue on android. On iOS the issue doesn't seem to be present.

On IOS once clicked it ain't stopping then i'm facing this issue

**Comment by valentinbld45** [comment]
Any updates for this feature?

**Comment by Asko-Dev** [comment]
any updates?

**Comment by MalikHamza007** [comment]
These Capacitor packages always have something going on.
